Update the README files and also add the default value for the temporary directory parameter. (#278)
IntelLabs/vdms: remote_function/README.md

## Setup
1. Copy the `remote_function` directory on the machine you want to run the remote server. Can be run on any location, independent of where VDMS is running. However, the location should be reachable from the machine that is running VDMS. You can also use `sparse-checkout` to only retrieve the `remote_function` directory from the VDMS repo.

> 2. Copy `resources` directory (located at the root of the repository) next to the `remote_function` directory.

Reviewer: For clarification, when you say `next to` should the `resources`  directory be copied into the `remote_function` directory? So resulting directory would be `remote_function/resources`? If this is correct, I suggest changing it to:
```suggestion
2. Copy `resources` directory (located at the root of the repository) into the `remote_function` directory.
```

Author: Hi Chaunté, the "resources" and "remote_function" directories should be siblings.
As they are siblings directories, could you please tell me if I should clarify that or if the use of "next" could confuse to the reader?
Thank you.

Reviewer: @rolandoquesada, I'm was a little confused with the `next to` term. When you say sibling directories, they should be in same parent directory? So say we have `parent/remote_function`, then the resources path should be `parent/resources`? If so, I suggest:
```suggestion
2. Copy `resources` directory (located at the root of the repository) into same directory as the `remote_function` directory.
```
What do you think?

Reviewer 2: The suggestion by @cwlacewe sounds good. 

Otherwise, you can include the `resources` directory copy instruction in Step 1 if it still sounds confusing. So you say, "Copy the `remote_function` and `resources` directories on the machine you want to run the remote server. Both should be in the same parent directory. It can be on any server, independent of where VDMS is running. However, the server should be reachable from the machine that is running VDMS."

## Detailed Instructions for new remote operation
We now provide an example to add a new operation `cardetect` as a remote operation that would work with VDMS. The `cardetect` operation detects cars in an image and creates a rectangle around all cars. This operation requires a pretrained model available in the form of `xml` file online.

1. Copy `remote_function` directory to your remote server machine. Say the address is `my.remote.server` and you copy the folder in the `home` directory.

> 2. Copy `resources` directory (located at the root of the repository) next to the `remote_function` directory. The folder structure you have now will look something like this;

Reviewer: Same comment as `Setup` section
